src/mavlink-router/mainloop.cpp

```diff
@@ -760,30 +772,50 @@ void Mainloop::_handle_pipe()
 
           // Parse each command
           // Command Format:
-          // Cmd UDP Name IP Port Eavesdropping
-          // e.g. add udp application 127.0.0.1 14532 0
+          // Cmd UDP Name IP Port Eavesdropping coalesce_bytes coalesce_ms coalesce_nodelay_ids
+          // e.g. add udp application 127.0.0.1 14532 0 1500 30 75,76,77
```

Comment:
parsing input and taking actions on it should always go into separate functions, coupled by a data structure representing the parse

here the "structure" used is a vector (discouraged, of course, but not asking to change it), so that would be the natural splitting point

also, are the message sets to be delivered without a delay a "per-endpoint" configuration thing?

Reply:
> are the message sets to be delivered without a delay a "per-endpoint" configuration thing?

Yes, per endpoint